toast-tk-web-agent/src/main/java/io/toast/tk/agent/web/rest/ToastAsyncHttpClientProvider.java

```diff
@@ -0,0 +1,42 @@
+package io.toast.tk.agent.web.rest;
+
+import com.google.common.base.Strings;
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import io.toast.tk.agent.config.AgentConfig;
+import org.asynchttpclient.AsyncHttpClientConfig;
+import org.asynchttpclient.DefaultAsyncHttpClient;
+import org.asynchttpclient.DefaultAsyncHttpClientConfig;
+import org.asynchttpclient.Realm;
+import org.asynchttpclient.proxy.ProxyServer;
+
+
+public class ToastAsyncHttpClientProvider implements Provider<DefaultAsyncHttpClient> {
+
+
+    private final AgentConfig config;
+
+    @Inject
+    public ToastAsyncHttpClientProvider(AgentConfig config){
+        this.config = config;
+    }
+
+    @Override
+    public DefaultAsyncHttpClient get() {
+        if(Boolean.valueOf(this.config.getProxyActivate()) == true){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1125)